measures/2018/measures-data.json

```diff
@@ -47227,6 +47227,492 @@
       "administrativeClaims"
     ]
   },
+  {
```

Comment:
@sarahnw @kalvinwang I've refactored the script to generate the CAHPS measures based on @kalvinwang suggestions.  Can you verify the generated CAHPS measures (ACO and non-ACO) are correct?

Reply:
Thanks for the review.  I added ACO321 as an additional measure in this PR.

Reply:
(Looks like sarah approved in the jira ticket)